lib/element.js

```diff
@@ -350,23 +350,7 @@ ElementArrayFinder.prototype.toElementFinder_ = function() {
  */
 ElementArrayFinder.prototype.count = function() {
   return this.getWebElements().then(function(arr) {
-    var list = arr.map(function(webElem) {
-      // Calling any method forces a staleness check
-      return webElem.isEnabled().then(function() {
-        return 1; // is present
-      }, function(err) {
-        if (err.code == webdriver.error.ErrorCode.STALE_ELEMENT_REFERENCE) {
-          return 0; // not present
-        } else {
-          throw err;
-        }
-      });
-    });
-    return webdriver.promise.all(list).then(function(presenceArr) {
-      return presenceArr.reduce(function(acc, isPresent) {
-        return acc + isPresent;
-      }, 0);
-    });
+    return arr.length;
```

Comment:
The only issue here is that the this number we return is not necessarily correct anymore.  I feel like this should be documented somewhere